### PR TITLE
To Do List Review

### PR DESCRIPTION
**Pull request by Johnadibe**
## To-Do List Review

- Used webpack to bundle JavaScript code.
- Used ES6 modules to write modular JavaScript.
- Rendered the created list of tasks dynamically on page load.
- Populated the Html list item element for each task.
- Implemented a function for adding a new task to an array.
- Implemented a function for deleting a task from an array.
- Implemented a function for editing task descriptions.
- Saved all To-Do List changes in local storage.
- Implemented a function for the "Clear all completed" button.
- Stored the updated array of items in local storage.
- Updated items object's value for completed key upon user actions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }
+  const list = {
+    description: userInput,
+    completed: false,
+    index: todos.length,
+  };
+  todos.push(list);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTodo();
+};
```
Comment (Outkarm) [inline src/index.js:48]: i would suggest you use comments in your code to make it easier to follow 😃

**Review by Outkarm — CHANGES_REQUESTED** [review]
Hi @Johnadibe ,
Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
image
 
Suggested changes
Check the comments under the review.
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Johnadibe** [comment]
Thank you Outkarm, I will make the suggested changes immediately.

File: src/index.js (hunk 1 above)
Comment (Johnadibe) [reply src/index.js:48]: Thanks for that😊

**Review by Outkarm — APPROVED** [review]
Hello [@Johnadibe](https://github.com/Johnadibe) 👋 ,

I am Johnson your reviewer for this PR 🧔‍♂️

Thank you for meeting all the project requirements good job! 🥳

Your project is complete! There is nothing else to say other than... it's time to merge it  
Congratulations! 🥳 🎉

image

 
Highlights
Great job setting up the correct linters ✔️
Professional documentation ✔️
Correct workflow ✔️
Meaningful commit messages ✔️
Met all the milestone requirements ✔️
[Optional] suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Outkarm in your question so I can receive the notification.

**Comment by Johnadibe** [comment]
Thank you Johnson😊

**Pull request merged**